render-test/runner.hpp

```diff
@@ -11,11 +14,14 @@ struct TestMetadata;
 
 class TestRunner {
 public:
-    TestRunner() = default;
-    explicit TestRunner(const std::string& testRootPath);
+    explicit TestRunner(mbgl::optional<Manifest> manifest_);
```

Comment:
explicit TestRunner(Manifest manifest_); // we cannot have TestRunner without a manifest